File: BocetoV2/DaoMock/EstadoAnimoDaoMock.java
import java.time.LocalDate;

public class EstadoAnimoDaoMock extends EstadoAnimoDao {
    public EstadoAnimoDaoMock() {
        super();
        EstadoAnimo estado = new EstadoAnimo();
        // Dato de prueba inicial
        estado.getRegistroAnimo().put(LocalDate.now(), IconoEstadoAnimo.BIEN);
        this.guardar(estado);
    }
}

File: App/AppV2/app/src/main/java/com/pmdm/planify/data/daomocks/EstadoAnimoDaoMock.kt
package com.pmdm.planify.data.daomocks

import com.pmdm.planify.data.mocks.EstadoAnimoMock
import com.pmdm.planify.models.IconoEstadoAnimo
import java.time.LocalDate

class EstadoAnimoDaoMock {
    // Simulamos la tabla de Estado de Ánimo
    val estadoAnimo = EstadoAnimoMock().apply {
        registroAnimo[LocalDate.now()] = IconoEstadoAnimo.BIEN
    }
}